@navikt/core/css/darkside/tag.darkside.css

```diff
@@ -1,194 +1,132 @@
 .navds-tag {
-  --__ac-tag-icon-size: 1.5rem;
-  --__ac-tag-icon-margin: -2px;
-
-  border: 1px solid;
-  border-radius: var(--a-border-radius-small);
+  --__axc-tag-icon-size: 1.5rem;
+  --__axc-tag-icon-margin: -2px;
+  --__axc-tag-border: ;
+  --__axc-tag-bg: ;
+  --__axc-tag-bg-strong: ;
+  --__axc-tag-text: ;
+  --__axc-tag-text-strong: ;
+
+  border-radius: var(--ax-border-radius-medium);
   display: inline-flex;
   align-items: center;
   justify-content: center;
-  padding: var(--a-spacing-05) var(--a-spacing-2);
+  padding: var(--ax-spacing-05) var(--ax-spacing-2);
   min-height: 2rem;
   line-height: 1;
-}
-
-.navds-tag--small {
-  min-height: 1.5rem;
-  padding: 0 var(--a-spacing-1-alt);
-
-  --__ac-tag-icon-size: 1.25rem;
-}
-
-.navds-tag--xsmall {
-  min-height: 1.25rem;
-  padding: 0 var(--a-spacing-1);
-
-  --__ac-tag-icon-size: 1rem;
-  --__ac-tag-icon-margin: -1px;
-}
-
-.navds-tag:has(.navds-tag__icon--left) {
-  gap: var(--a-spacing-05);
-}
 
-.navds-tag__icon--left {
-  font-size: var(--__ac-tag-icon-size);
-  margin-inline-start: var(--__ac-tag-icon-margin);
-  display: flex;
-}
-
-.navds-tag--error {
-  border-color: var(--ac-tag-error-border, var(--a-border-danger));
-  background-color: var(--ac-tag-error-bg, var(--a-surface-danger-subtle));
-  color: var(--ac-tag-error-text, var(--a-text-default));
-}
-
-.navds-tag--error-filled {
-  border-color: transparent;
-  background-color: var(--ac-tag-error-filled-bg, var(--a-surface-danger));
-  color: var(--ac-tag-error-filled-text, var(--a-text-on-danger));
+  &:has(.navds-tag__icon--left) {
+    gap: var(--ax-spacing-05);
+  }
 }
 
-.navds-tag--error-moderate {
-  border-color: transparent;
-  background-color: var(--ac-tag-error-moderate-bg, var(--a-surface-danger-moderate));
-  color: var(--ac-tag-error-moderate-text, var(--a-text-default));
+.navds-tag--info,
+.navds-tag--alt3 {
```

Comment:
alt3 will not get a replacement?

Reply:
No, info was `light-blue` before, but deep-blue now so they are currently the same. The designers are potentially discussing adding a "teal"-color to the system tho.